types/utils.go

```diff
@@ -142,15 +142,3 @@ func LogDeferred(logger log.Logger, f func() error) {
 		logger.Error(err.Error())
 	}
 }
-
-// SliceContains implements a generic function for checking if a slice contains
-// a certain value.
-func SliceContains[T comparable](elements []T, v T) bool {
```

Comment:
on the backport we either have to add this to the store package or add it back here